namer/namer.cc

```diff
@@ -271,15 +269,13 @@ class NameInserter {
         return klass;
     }
 
-    bool handleNamerRewriter(core::MutableContext ctx, unique_ptr<ast::ClassDef> &klass,
+    void handleNamerRewriter(core::MutableContext ctx, unique_ptr<ast::ClassDef> &klass,
                              unique_ptr<ast::Expression> &line) {
-        if (addAncestor(ctx, klass, line)) {
-            return true;
-        }
+        addAncestor(ctx, klass, line);
```

Comment:
One point: the code below never used to run if `addAncestor` fired for any condition.

Not sure that it matters from a correctness standpoint, but it does mean we're technically doing more work.